app/src/views/DrefApplicationExport/PgaExport/index.tsx

```diff
@@ -0,0 +1,248 @@
+                    />
+                    <TextOutput
+                        className={styles.drefTable}
+                        label={strings.imminentIFRCSigned}
+                        value=""
+                        invalidText
+                        strongLabel
+                        withoutLabelColon
+                    />
+                    <TextOutput
+                        className={styles.signature}
+                        label={strings.imminentSignature}
+                        value=""
+                        invalidText
+                        withoutLabelColon
+                        strongLabel
+                    />
+                    <TextOutput
+                        className={styles.signature}
+                        label={strings.imminentSignature}
+                        value=""
+                        invalidText
+                        withoutLabelColon
+                        strongLabel
+                    />
+                    <TextOutput
+                        className={styles.signature}
+                        label={strings.imminentPrintedSignatory}
+                        value=""
+                        invalidText
+                        withoutLabelColon
+                        strongLabel
+                    />
+                    <TextOutput
+                        className={styles.signature}
+                        label={strings.imminentPrintedSignatory}
+                        value=""
+                        invalidText
+                        withoutLabelColon
+                        strongLabel
+                    />
+                    <TextOutput
+                        className={styles.signature}
+                        label={strings.imminentTitle}
+                        value=""
+                        invalidText
+                        withoutLabelColon
+                        strongLabel
+                    />
+                    <TextOutput
+                        className={styles.signature}
+                        label={strings.imminentTitle}
+                        value=""
+                        invalidText
+                        withoutLabelColon
+                        strongLabel
+                    />
+                    <TextOutput
+                        className={styles.signature}
+                        label={strings.imminentDate}
+                        value=""
+                        invalidText
+                        withoutLabelColon
+                        strongLabel
+                    />
+                    <TextOutput
+                        className={styles.signature}
+                        label={strings.imminentDate}
+                        value=""
+                        invalidText
+                        withoutLabelColon
+                        strongLabel
+                    />
+                </div>
+            </Container>
```

Comment:
These also doesn't seem to be using `invalidText`, `value` and `className` props for it's intended, let's create a separate component if necessary